src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,56 @@
+import classnames from 'classnames';
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Loader } from '../Loader/Loader';
+
+type Props = {
+  todos: Todo[],
+  isLoading: boolean
+};
+
+export const TodoList: React.FC<Props> = ({ todos, isLoading }) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
+
+      {todos.map(({ title, id, completed }) => {
+        return (
+          <div
+            key={id}
+            data-cy="Todo"
+            className={classnames(
+              'todo',
+              {
+                completed: completed === true,
```

Comment:
Here you can avoid explicit equality check, `completed` is already a boolean, it is either true of false on its own